website/src/pages/index.js

```diff
@@ -127,8 +136,29 @@ function Home() {
             </div>
           </section>
         )}
+        {showcase && showcase.apps.length > 0 && (
+          <section className='container'>
+            <div className={clsx('row', styles.showcaseTitleContainer)} >
+              <div className={styles.divider} style={{ marginRight: 10 }} />
+              <h1 className={styles.showcaseTitle}>{`See what you can build\nwith Wave`}</h1>
+              <div className={styles.divider} style={{ marginLeft: 10 }} />
```

Comment:
Let's also add a note under the h2:

The following are just a few of hundreds of Wave apps built within H2O.ai. The most mind-blowing thing is that all **presented apps were built by our data science and machine learning people**, no software engineer necessary.

Reply:
Good idea. What do you mean by note? Just simple `<p>` tag or does it need some special formatting?

Reply:
`p` is fine